tests/testthat/test-quit_if_no_pacta_relevant_data.R

```diff
@@ -0,0 +1,51 @@
+test_that("`quit_if_no_pacta_relevant_data()` quits if no ABCD in non-interactive", {
```

Comment:
NIT: In my tests, I usually omit the function name (`quit_if_no_pacta_relevant_data()`), since it is assumed that any test in the file `test-quit_if_no_pacta_relevant_data.R` relates to that function). 

That's why you will see tests [here](https://github.com/2DegreesInvesting/r2dii.analysis/blob/main/tests/testthat/test-target_market_share.R) with names like "w/ bad data, errors with informative message"). 

This could, for example, just be "quits if no ABCD in non-interactive session". Just saves clutter when writing many tests.